tests/cypress/integration/usersTest.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { users } from '../page-object/users.page'
+
+describe('navigation to user', () => {
+    it('navigates to the users page successfully', function () {
+        users.goTo()
```

Comment:
No assertion how do we know it is successful?

Reply:
this isn't a full spec, it just demonstrates how it looks. 
ideally this would be followed by additional actions and assertions, I was hoping to have an actual relevant use-case for sitemap, but as of now i didn't get one yet, so this irrelevant example stays.

Reply:
Is it possible to show a full example to SEO portion? 
Or reference to an existing test to show as an example

Reply:
navigating to the SEO part of a site?
sure, I'll add it.